docs/changelog-template.md

```diff
@@ -0,0 +1,52 @@
+# Changelog
```

Comment:
@damacus brought up a good point on one of my PRs. Do we want to have the project name in the top line or just "Changelog"?

Reply:
I think `Changelog` is fine. Looking at supermarket it's actually sort of redundant to have the cookbook name in the changelog file:

![image](https://user-images.githubusercontent.com/19351306/67711254-99f32880-f98f-11e9-9ed1-20c084d74399.png)